app/javascript/components/batch_update/components/PriceInput.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import TextInput from './TextInput'
+
+const PriceInput = ({ minPrice, maxPrice, updateState }) => {
+  const onSetMin = value => updateState('minPrice', value)
+  const onSetMax = value => updateState('maxPrice', value)
+
+  return (
+    <div>
+      {minPrice == null ? (
+        <TextInput
+          placeholder="Minimum Price"
+          value={minPrice}
+          onEnter={onSetMin}
+          numeric
+        />
+      ) : (
+        <div />
```

Comment:
Is this empty div necessary? This could just be `null`

Or even simpler with &&: 

```js
{minPrice == null && (<TextInput ... />)}
```